### DISCUSSION TITLE
support tickets / custom invoice?

### DISCUSSION THREAD

**Discussion by Spikes-Coding** [pimcore/pimcore#10421] [Q&A / Support]
is there a plugin for support tickets / custom invoice?

**Comment by WebSteR102** [comment]
You could try CoreShop if you are building eCommerce solution.

If you are a beginner at Pimcore and need more information I recommend you start [here](https://factory.dev/pimcore-knowledge-base/how-to/pimcore-for-beginners).

Good luck
